index.html

```diff
@@ -95,6 +93,36 @@ <h2>Facebook 360</h2>
           </div>
       </section>
       <!-- First section ends -->
```

Comment:
- [ ] OPTIONAL
There are comments found in your code on the repository. Kindly endeavor to remove **all** comments from your code before pushing it to the repository so as to keep your code cleaner.